eng/scripts/smoketest/Get-Package-Version.ps1

```diff
@@ -56,7 +56,7 @@ function Get-SmokeTestPkgProperties
     foreach ($pkg in $newPackages) {
         $pkgVersion = [AzureEngSemanticVersion]::ParseVersionString($pkg.Version)
         $pkgVersion.IsPreRelease = $false
```

Comment:
```suggestion
        $pkgVersion.PrereleaseLabel = ""
```

Reply:
For reference this was broken with @hallipr change to SemVer https://github.com/Azure/azure-sdk-for-net/pull/47932/files#diff-6daee72da4bcdb8ca9c1a437278ebb3e844868cc59a6a7ef4227e2381e64b2b2L139